examples/applications/plot_prediction_latency.py

```diff
@@ -232,7 +232,7 @@ def plot_n_features_influence(percentiles, percentile):
     fig, ax1 = plt.subplots(figsize=(10, 6))
     colors = ["r", "g", "b"]
     for i, cls_name in enumerate(percentiles.keys()):
-        x = np.array(sorted([n for n in percentiles[cls_name].keys()]))
+        x = np.array(sorted(list(percentiles[cls_name].keys())))
```

Comment:
In this case, I do not think the `list` is required either:

```suggestion
        x = np.array(sorted(percentiles[cls_name].keys()))
```

Reply:
yes, you are right i have changed the file now you can push on merge on main branch
thank you